test/e2e/applicationset_git_generator_test.go

```diff
@@ -20,6 +22,16 @@ import (
 	"github.com/argoproj/argo-cd/v3/test/e2e/fixture/applicationsets/utils"
 )
 
+func randStr(t *testing.T) string {
+	t.Helper()
+	bytes := make([]byte, 16)
```

Comment:
16 bytes because it doesn't really matter. We just need random content for the test.